<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1425</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StockView* 1.0.0</string>
  </property>
  <widget class="QWidget" name="Frame">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="Grapher_Layout" stretch="1,0,0,0">
      <item>
       <widget class="ChartWidget" name="StockView_Chart" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="GraphInput_Layout">
        <item>
         <widget class="QLabel" name="Instructions_Label">
          <property name="text">
           <string>Ticker Symbol(s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="TickerSymbols_LineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="GraphStocks_Button">
          <property name="text">
           <string>Graph</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Start Date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="StartDate_lineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>End Date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="EndDate_lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="DataFile_LineEdit">
        <property name="text">
         <string>Current Data File:</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="PythonRunner_Layout" stretch="0,0,0,0,0,1,0">
      <item>
       <widget class="QLabel" name="FileSelect">
        <property name="text">
         <string>Select a file</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="FileSelect_Layout" stretch="1,0">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLineEdit" name="FileSelect_LineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="FileSelector_Button">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="InputArguments">
        <property name="text">
         <string>Additional input arguments</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="InputArguments_LineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="ConsoleOutput">
        <property name="text">
         <string>Console Output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="ConsoleOutput_TextBrowser"/>
      </item>
      <item>
       <widget class="QPushButton" name="Run_Button">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChartWidget</class>
   <extends>QWidget</extends>
   <header>ChartWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
